### ISSUE TITLE
Basleine Training

### ISSUE THREAD

**Issue by DivyaIITP** [apeterswu/RL4NMT#6]
In the paper you maintained about verifying the baseline reward approach and the baseline reward estimator was pretrained. But i couldn't see the pretraining code !. Can you help me with the pretraining code  ??
